lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommDriver.cs

```diff
@@ -51,9 +51,11 @@ public sealed class GroupCommDriver : IGroupCommDriver
         private readonly string _driverId;
         private readonly string _nameServerAddr;
         private readonly int _nameServerPort;
+        private readonly int _defaultGroupNumberOfTasks;
```

Comment:
Julia in general I do not understand the need of doing passive default group creation for fault tolerance. Can you explain?